src/config/SettingsConfig.cc

```diff
@@ -48,10 +50,14 @@ struct SettingsParser : rapidsax::NullHandler
 		BINDINGS,
 		BIND,
 		UNBIND,
+		SHORTCUTS,
+		SHORTCUT,
 		END
 	} state = START;
 	Setting currentSetting;
 	HotKey::Data currentBind;
+	int currentShortcutIndex;
```

Comment:
Detail: both this `int` and the one on line 40 (the first field of the `pair`) should probably be `ShortcutIndex`?